### ISSUE TITLE
How to pass a externref to wasm in command line ?

### ISSUE THREAD

**Issue by Zhangyx24** [bytecodealliance/wasmtime#4423]
### Question

I want to excute the following wasm program : 
[test.txt](https://github.com/bytecodealliance/wasmtime/files/9077292/test.txt)
,but how to pass the externref parameter?

**Comment by bjorn3** [comment]
I don't think you can. What would you put in the externref anyway? It is an opaque reference to a value of any type in the embedder.

**Comment by Zhangyx24** [comment]
Do you mean that it can only be used in the host high-level language api ?
Could you please give me a simple example?

**Comment by bjorn3** [comment]
I think so. Here is an example of storing a string in an `ExternRef`: https://github.com/bytecodealliance/wasmtime/blob/6c70428735d87314a7568542939e1e74f79f7871/examples/externref.rs#L22

**Comment by Zhangyx24** [comment]
Thanks a lot !
